Specs/LlamaKit/0.1.1/LlamaKit.podspec.json

```diff
@@ -10,8 +10,8 @@
   },
   "social_media_url": "http://twitter.com/cocoaphony",
   "platforms": {
-    "ios": "8.0",
-    "osx": "10.10"
+    "ios": "7.0",
```

Comment:
I don't think this is right, frameworks required to integrate Swift are not available on iOS 7.0.